test/WebJobs.Script.Tests/Configuration/HostJsonFileConfigurationSourceTests.cs

```diff
@@ -20,7 +20,7 @@ namespace Microsoft.Azure.WebJobs.Script.Tests.Configuration
     public class HostJsonFileConfigurationSourceTests
     {
         private readonly string _hostJsonWithBundles = "{\r\n  \"version\": \"2.0\",\r\n  \"extensionBundle\": {\r\n    \"id\": \"Microsoft.Azure.Functions.ExtensionBundle\",\r\n    \"version\": \"[2.*, 3.0.0)\"\r\n  }\r\n}";
-        private readonly string _hostJsonWithWorkFlowBundle = "{\r\n  \"version\": \"2.0\",\r\n  \"extensionBundle\": {\r\n    \"id\": \"Microsoft.Azure.Functions.ExtensionBundle.Workflows\",\r\n    \"version\": \"[2.*, 3.0.0)\"\r\n  }\r\n}";
+        private readonly string _hostJsonWithWorkFlowBundle = "{\r\n  \"version\": \"2.0\",\r\n  \"extensionBundle\": {\r\n    \"id\": \"Microsoft.Azure.Functions.ExtensionBundle.Workflows\",\r\n    \"version\": \"[1.*, 2.0.0)\"\r\n  }\r\n}";
```

Comment:
I don't see this test changing to run as LogicApp so why did you change the default to the older version?

Reply:
Not sure what you mean by that

Reply:
The test was already running as a logic app. I had changed this in error in the previous change.